packages/d/directxtk/xmake.lua

```diff
@@ -0,0 +1,59 @@
+function split(s, delimiter)
+    result = {};
+    for match in (s..delimiter):gmatch("(.-)"..delimiter) do
+        table.insert(result, match);
+    end
+    return result;
+end
+
+
+package("DirectXTK")
+
+    set_homepage("https://github.com/Tessil/hopscotch-map")
+    set_description("This package contains the \"DirectX Tool Kit\", a collection of helper classes for writing Direct3D 11 C++ code for Universal Windows Platform (UWP) apps for Windows 10, Xbox One, and Win32 desktop applications for Windows 7 Service Pack 1 or later.")
+
+    set_urls("https://github.com/microsoft/DirectXTK/archive/$(version).zip",
+             "https://github.com/microsoft/DirectXTK.git",
+             {version = function (version)
+                local versions = {
+                    ["20.9.0"] = "sept2020"
+                }
+
+                return versions[version]
+            end})
+
+    add_versions("20.9.0", "9d5131243bf3e33db2e3a968720d860abdcbbe7cb037c2cb5dd06046d439ed09")
+
+    on_install("windows", function (package)
+        local configs = {}
+        
+        local vs_sdkver = get_config("vs_sdkver")
+        if vs_sdkver then
+            split_version = {};
```

Comment:
Using Lua patterns would be an improvement, like `string.match("10.0.19041.0", "%d+%.%d+%.(%d+)%.?%d*")`